packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageBox.tests.js

```diff
@@ -107,6 +138,25 @@ QUnit.module('MessageBox', moduleConfig, () => {
             assert.strictEqual(onMessageSendStub.callCount, 0);
         });
 
+        QUnit.test('should not be fired on enter key if the textarea input does not contain a value (excluding spaces)', function(assert) {
```

Comment:
I expect to see test with checking arguments of MessageSend event. Something like this https://github.com/Zedwag/DevExtreme/blob/chatEnterKbn__24_2/packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageBox.tests.js#L189, only for Enter key press

Reply:
[commit](https://github.com/DevExpress/DevExtreme/pull/28003/commits/d11560dd6363390ad38ad1984acdf0a765acc8c1)